src/app/utils/DiscordUtils.ts

```diff
@@ -154,13 +155,29 @@ const DiscordUtils = {
             await commandContext.send({ content: content, ephemeral: true });
             // await commandContext.delete();
         } else {  // This was a reaction or a DB event
-            await toUser.send(content);
+            try {
+                await toUser.send(content);
+            } catch (e) {
+                throw new AuthorizationError(
+                    `Thank you for giving bounty commands a try!\n` +
+                    `It looks like bot does not have permission to DM you.\n` +
+                    'Please give bot permission to DM you and try again.'
+                )
+            }
         }
     },
 
     // Send a notification to an interested party (use a DM)
     async activityNotification(content: string, toUser: GuildMember): Promise<void> {
-        await toUser.send(content);
+        try {
+            await toUser.send(content);
+        } catch (e) {
+            throw new AuthorizationError(
```

Comment:
This message is actually going to a 3rd party who is just being notified. The error message should go to the person who executed the bounty command or reaction.

Reply:
Got it. Sending an Authorization error will not be appropriate here. However, we do need to catch it because we do not want the app to crash when 3rd party might have disabled DMs from the server. 

Publishing it to the default channel will be risky because there might be private messages in these